platform/administer/templates/overview.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: Overview
+sidebar_label: Overview
+sidebar_position: 1
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import Flow, { Step } from "@site/src/components/Flow";
+import Button from "@site/src/components/Button";
+import Label from "@site/src/components/Label";
+import NavStep from "@site/src/components/NavStep";
+
+
+# Overview
+
+vCluster templates define reusable configurations for creating virtual Kubernetes clusters. They allow platform teams to standardize cluster creation while giving users a controlled way to customize certain settings. Templates are implemented using Kubernetes custom resources (`VirtualClusterTemplate`) and can be managed with tools such as `kubectl`, Helm, or ArgoCD.
+
+Each template defines the base configuration for a virtual cluster, including the Kubernetes version, control plane type, networking mode, storage settings, and sync behavior. Sync settings control which Kubernetes resources are synchronized between the virtual cluster and the host cluster—such as Service, Secret, ConfigMap, or custom resources. Templates can use sync rules to import host resources, export virtual cluster resources, or customize synchronization at the namespace or cluster scope.
+
+To support secure and consistent self-service, templates allow administrators to define which settings users can modify through parameters. For example, you can expose only a few fields—such as the number of replicas or a set of approved storage classes—while keeping the rest of the configuration fixed. This helps reduce the risk of misconfiguration, but also enables flexibility.
+
+Templates can also include preinstalled apps, such as monitoring agents or ingress controllers, using Helm charts or Kubernetes manifests. Additional Kubernetes resources like ConfigMaps, RBAC, or NetworkPolicies can be applied to either the host cluster or the virtual cluster itself.
+
+Because templates can be versioned, they support upgrade workflows. When a template is updated, those changes can be propagated to any virtual clusters based on it. This allows teams to update cluster configurations and installed apps consistently across environments.
```

Comment:
Maybe update to read “ those changes can be manually or automatically propagated”

Suggestion, “update cluster configurations and installed apps consistently across a fleet of virtual clusters”